src/app/dashboard/components/dashboard.component.html

```diff
@@ -2,42 +2,57 @@
           <mat-label>Filter</mat-label>
-          <mat-select [formControl]="filter" class="itemborder" >
+          <mat-select [formControl]="filter" class="itemborder">
             <mat-option value="myItems">View Mine Only</mat-option>
             <mat-option value="showAllOrganization">View All Organization</mat-option>
           </mat-select>
         </mat-form-field>
-        <button mat-button mat-flat-button color="primary" class="btn cce-btn editbutton" (click)="showTrashClick(true)" *ngIf="!showTrash && isWeb">Edit</button>
-        <button mat-button mat-flat-button  class="btn cce-btn donebutton" (click)="showTrashClick(false)" *ngIf="showTrash && isWeb">Done</button>
+        <button
+          mat-button
+          mat-flat-button
+          color="primary"
+          class="btn cce-btn editbutton"
+          (click)="showTrashClick(true)"
+          *ngIf="!showTrash && isWeb"
+        >
+          Edit
+        </button>
+        <button mat-button mat-flat-button class="btn cce-btn donebutton" (click)="showTrashClick(false)" *ngIf="showTrash && isWeb">
+          Done
+        </button>
       </div>
     </div>
 
-    <div fxLayout="row" fxLayoutAlign="space-between center">
-           
-    </div>
+    <div fxLayout="row" fxLayoutAlign="space-between center"></div>
     <div *ngIf="!vm.loading; else inprogress">
       <div *ngIf="vm.needs && vm.needs.length > 0; else noneeds">
         <div *ngFor="let group of vm.needs">
-          <p class="group-header" *ngIf="vm.filterState === 'showAllOrganization'">{{group.createdBy}}</p>
+          <p class="group-header" *ngIf="vm.filterState === 'showAllOrganization'">{{ group.createdBy }}</p>
           <div *ngFor="let request of group.items" class="row-item">
             <div fxLayout="row" fxLayoutAlign="start center">
-              <mat-list-item (swipeleft)="toggleRowState(request, 'rowOpen')"
+              <mat-list-item
+                (swipeleft)="toggleRowState(request, 'rowOpen')"
                 (swiperight)="toggleRowState(request, 'rowClosed')"
-                [routerLink]="['/dashboard-item', 'request', request.itemId]" class="contentborder dashboard-list-item">
-                <div class="swipe-container"
-                  [@.disabled]="request.status !== status.OrderCancelled && request.status !== status.OrderFulfilled">
-                  <div class="dashboardlistitem" [@rowAnimate]="request.rowState"
+                [routerLink]="['/dashboard-item', 'request', request.itemId]"
+                class="contentborder dashboard-list-item"
+              >
+                <div
+                  class="swipe-container"
+                  [@.disabled]="request.status !== status.OrderCancelled && request.status !== status.OrderFulfilled"
+                >
+                  <div class="dashboardlistitem" [@rowAnimate]="request.rowState">
+                    <mat-label class="contentlabel"
+                      >{{ request.name }}<br />
```

Comment:
Can we keep tag elements to a same line as much as possible, breaking every line makes it hard to read where the tags end and start...

Reply:
Hm it must be a prettier thing. I'll see if there is a fix  for that.